constants.py

```diff
@@ -605,3 +605,23 @@
 kPhotonvisionCameraPixelDimensions = (960, 720)
 kApriltagWidth = 8.125 * kMetersPerInch
 kApriltagHeight = kApriltagWidth
+
+# IntakeMotor Motors
+kIntakeCANID = 3
+kIntakePIDSlot = 0
+kIntakePGain = 0.0001
+kIntakeIGain = 0
+kIntakeDGain = 0
+kIntakeMotorPercent = 0.7
+"""70%"""
+
+kCubeLoadedKey = "cubeInPos"
```

Comment:
i would tweak these into the `intake/` nettables folder for clarity, you are already using that for `kIntakeStateKey`